CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.8] - 2017-11-01
+### Fixed
```

Comment:
Sorry for annoying but can you please add an empty line between version number and `Fixed` - to keep the same style for the whole file.